tests/src/test/scala/cats/tests/OneAndSuite.scala

```diff
@@ -1,10 +1,7 @@
-package cats
-package tests
-
 import cats.kernel.laws.discipline.{EqTests, SemigroupTests}
 
-import cats.instances.stream._
-import cats.data.{NonEmptyStream, OneAnd}
+import cats.data.OneAnd
+import cats.`scala-2.13+`.data.NonEmptyLazyList // TODO how to import / does it even belong there?
```

Comment:
forgot to mention that we should just move these NonEmptyStream tests into 2.12-/cats/tests/NonEmptyStreamSuite.scala instead of changing them.